platform/config/cloud/standard/cloud_template_user_provided.json

```diff
@@ -0,0 +1,21 @@
+{
+    "cloud": {
+        "provider": "aws",
+        "version": 1,
+        "configure": {
+            "cluster_user": "",
+            "cluster_size": "user_provided",
+            "cluster_type": "standard",
```

Comment:
cluster type, region, placement, axsys_node_type, axuser_node_type, can be removed. node counts can be set to -1 as an indication of invalidness (user provides the cluster, we don't care).

Reply:
Are you sure removing the options won't break anything? This file was created based on the template used for other small/medium/etc. clusters.

Reply:
nope. These can be empty as we are not touching kube-up side. And I would assume ADC will have a proper mechanism to admit all if this is user provisioned cluster.